packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -453,7 +535,16 @@ const GlobalHeader = ({ className, activeSite }) => {
                 display: flex;
               `}
             >
-              <SplitTextButton />
+              <SplitTextButton
+                style={{
```

Comment:
If possible, we could move this over to the `css` prop provided by emotion. Similar to how we moved the `SearchInput` over.